rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -73,6 +73,7 @@ rabbitmq_ulimit: 65535
 # since memory utilization is primarily driven by the objects placed in memcached.
 memcached_connections: 16384
 
+#TODO (evrardjp): Remove this when ocata
 # Keystone overrides
 keystone_token_driver: "sql"
```

Comment:
The alternative would be to have, in OSA, a sql fallback in all cases, and drop the memcache. I have a commit somewhere for that.